src/styles/shelley/tabs.st.css

```diff
@@ -0,0 +1,60 @@
+/* tabs.st.css */
+/* = Shelley project scope and vars */
+@st-import Shelley, [
+    --color-ui-inverse,
+    --color-ui-raw,
+    --color-accent-1
+  ] from "./project.st.css";
+/* Component API */
+@st-import Tabs from "../../components/Tabs/tabs.st.css";
+
+/* = This is how we can scope the styles in Stylable. */
+@st-scope Shelley {
+
+  /* Underline width or height is calculated in the component level */
+  Tabs::tabSelection {
+    background-color: var(--color-accent-1);
+  }
+
+  /* Horizontal tabs underline */
+  Tabs:orientation(horizontal)::tabSelection {
+    left: 0;
+    height: 2px;
```

Comment:
These comments are a bit confusing to me and a bit misleading.
Do we want to say "Base underline width or height is calculated in the component level"?
Only asking because in the CSS declaration below we set the height to 2px.